huey_monitor/admin.py

```diff
@@ -23,11 +46,21 @@ def get_queryset(self, request):
 
 @admin.register(TaskModel)
 class TaskModelAdmin(admin.ModelAdmin):
+    request_parameters={} # for storing the request GET parameters
+    
+    def changelist_view(self, request, *args, **kwargs):
+        self.request_parameters = request.GET
+        return super().changelist_view(request, *args, **kwargs)
+    
     def get_changelist(self, request, **kwargs):
         return TaskModelChangeList
 
     def column_name(self, obj):
-        qs = TaskModel.objects.filter(parent_task_id=obj.pk).order_by('-create_dt')
+        if 'is_maintask' in self.request_parameters:
```

Comment:
The "work-a-round" with `self.request_parameters` is ugly, isn't it?

Is there a `self.request` or a other way to solve this here?